### ISSUE TITLE
Flameshot won't start no error

### ISSUE THREAD

**Issue by Marc-AntoineH** [flameshot-org/flameshot#1261]
Hi,

yesterday flameshot was working and now I try to launch it, and its not on the system tray like it should be.

```
$ps -ef | grep flameshot
4308    4037  0 07:41 ?        00:00:00 flameshot
16382   16055  0 07:52 pts/1    00:00:00 grep --color=auto flameshot
```
it is running as a process.

**Comment by Marc-AntoineH** [comment]
FIXED:  I did try to reboot, or uninstall and install Flameshot. What I did to make it work, I remove it from Startup Application, then I kill the process and launch it again. Everything is working now, the Flameshot iconn is on the sys Tray.

I think the Startup App. is causing issue.


I am on Pop Os Focal - GNOME

**Comment by mmahmoudian** [comment]
@Marc-AntoineH i don't think this a Flameshot bug/issue. In case you also agree, I suggest that we close this issue.

**Comment by slovdahl** [comment]
I don't recall exactly when (maybe when I upgraded to Ubuntu 20.04), but this has also started happening to me. Having it in Startup Applications with GNOME 3 causes it to remain as a process in the background, but it's not shown in the task bar. Killing it manually and starting after that makes it turn up in the task bar.

This is all I can find in the log during boot related to flameshot:
```
mar 07 21:09:26 desk whoopsie-upload-all[3116]: /var/crash/_usr_bin_flameshot.1000.crash already marked for upload, skipping
```

**Comment by mmahmoudian** [comment]
@slovdahl may I suggest moving away from Gnome 😉
But on the serious note, if you still have this issue, please open a new issue and provide your detailed system info so that we can try to reproduce and solve it. It would also be interesting to see if you can replicate the same effect in a new Ubuntu VM.

**Comment by slovdahl** [comment]
> But on the serious note, if you still have this issue, please open a new issue and provide your detailed system info so that we can try to reproduce and solve it. It would also be interesting to see if you can replicate the same effect in a new Ubuntu VM.

Thanks for the suggestion, I tried it in a VM with a fresh Ubuntu 20.04. It looks like this may a problem with the version that Ubuntu 20.04 bundles. By installing it with `apt install flameshot` and adding `flameshot` to the startup applications, I got the problem reproduced. Ubuntu 20.04 has version 0.6.0.

After that I realized that it's also available as a snap, and after installing the snap instead it worked. :tada: The snap version was 0.9.0.rc2. Confirmed on the host as well that replacing the `apt` package with the snap solved the problem.
